### ISSUE TITLE
CONSOLE ERROR SyntaxError: Invalid regular expression: invalid flags

### ISSUE THREAD

**Issue by OoLunar** [Boscop/web-view#176]
I'm... Not really sure what's going on here. I'm currently trying to select a few local files, and present them in the GUI. Here's what I got:
My Cargo dependencies 
```toml
[dependencies]
web-view = { version = "0.6.2" }
tinyfiledialogs = "3.3.9"
dirs = "2.0.2"
```
main.rs:
```rust
use web_view::*;

fn main() {
    let html_content = std::fs::read_to_string("src/HTML/index.html").unwrap();
    web_view::builder()
        .title("Image Viewer")
        .content(Content::Html(html_content))
        .size(500, 600)
        .resizable(true)
        .debug(true)
        .user_data(())
        .invoke_handler(|webview, arg| match arg {
            "open_images" => {
                let files = tinyfiledialogs::open_file_dialog_multi("Choose your images:", dirs::home_dir().unwrap().to_str().unwrap(), Some((&["*.bmp", "*.gif", "*.jpg", "*.jpeg", "*.png", "*.tif", "*.tiff", "*.webm"], "Images"))).unwrap();
                let mut return_images: String = String::new();
                files.into_iter().for_each(|x: String| return_images.push_str(&format! {"{}\n", x}));
                let return_images = return_images.trim();
                println!("{}", return_images);
                webview.eval(&format!("handle_images({})", return_images)).unwrap();
                Ok(())
            }
            _ => unimplemented!(),
        })
        .run()
        .unwrap();
}
```
Here's the output:
```
/home/lunar/Pictures/Screenshot from 2020-03-23 12-25-25.png
/home/lunar/Pictures/Screenshot from 2020-03-23 14-24-19.png
data:text/html,%3C%21DOCTYPE%20html%3E%0A%0A%3Chtml%20lang%3D%22en%22%3E%0A%0A%3Chead%3E%0A%20%20%20%20%3Ctitle%3EImage%20Viewer%20-%20ForSaken%20Borders%3C%2Ftitle%3E%0A%20%20%20%20%3Cmeta%20charset%3D%22UTF-8%22%3E%0A%20%20%20%20%3Cmeta%20name%3D%22viewport%22%20content%3D%22width%3Ddevice-width%2C%20initial-scale%3D1%22%3E%0A%20%20%20%20%3Cmeta%20name%3D%22author%22%20content%3D%22Lunar%22%3E%0A%20%20%20%20%3Clink%20href%3D%22https%3A%2F%2Ffiles.forsaken-borders.net%2Fimages%2FForSaken%2520Borders.png%22%20rel%3D%22icon%22%20type%3D%22image%2Fpng%22%3E%0A%20%20%20%20%3Clink%20href%3D%22index.css%22%20rel%3D%22stylesheet%22%20type%3D%22text%2Fcss%22%20charset%3D%22UTF-8%22%3E%0A%3C%2Fhead%3E%0A%0A%3Cbody%3E%0A%20%20%20%20%3Cdiv%20id%3D%22temp%22%3EHello%20world%21%3C%2Fdiv%3E%0A%3C%2Fbody%3E%0A%3Cscript%20type%3D%22application%2Fjavascript%22%20charset%3D%22UTF-8%22%20src%3D%22index.js%22%3E%3C%2Fscript%3E%0A%3Cscript%3E%0A%20%20%20%20external.invoke%28%22open_images%22%29%3B%0A%3C%2Fscript%3E%0A%3C%2Fhtml%3E:1: CONSOLE ERROR SyntaxError: Invalid regular expression: invalid flags
```
Using Ubuntu 20.04, Webkit2gtk 0.29.1

**Comment by OoLunar** [comment]
Forgot to clarify this, but what it outputs is just my index.html. The JS function that's being called on `webview.eval(&format!("handle_images({})", return_images)).unwrap();` never gets executed.

**Comment by edko99** [comment]
I think you are missing the quotes that should surround, and the commas that should separate, the arguments to the handle_images() function call. Maybe you can print the results of the format, and see what JS code will get executed by eval.

**Comment by OoLunar** [comment]
When I quote the arguments as such
```rust
webview.eval(&format!("handle_images(\"{}\")", return_images)).unwrap();
```
, I'll get a new error `CONSOLE ERROR SyntaxError: Unexpected EOF`. I'm just looking for a way to allow the user to access the filesystem, and get a few filenames from it.

**Comment by OoLunar** [comment]
Found the issue!

**When passing the parameters to the JS functions, the string had a new line in it.** This was because I couldn't figure out how to convert a Vec<String> into an array of strings to pass onto the JS Function. My solution to that was just joining the vector into one string, then splitting by newlines. I replaced the newlines with commas, which seems to have gotten the JS function to start executing. 

Something I noticed however, is that it wouldn't execute the function if it was in `index.js` instead of `index.html`. This gave the error `CONSOLE ERROR ReferenceError: Can't find variable: handle_images`, which can be severely misleading if you have a large project going.
Thanks for the help, have a wonderful day!
